src/sonic-build-hooks/hooks/apt-get

```diff
@@ -11,20 +11,17 @@ if [ -z "$REAL_COMMAND" ]; then
 fi
 
 INSTALL=$(check_apt_install "$@")
-COMMAND_INFO="Locked by command: $REAL_COMMAND $@"
```

Comment:
Why we need to remove it?

Reply:
dpkg can't use this kind of lock. because of the feture of 'eatmydata'.
eatmydata also hooked dpkg command. So, we can't hook dpkg again.
It didn't resolve dpkg lock issue.